ipatests/test_integration/test_installation_client.py

```diff
@@ -71,10 +71,104 @@ def test_client_install_with_ssh_trust_dns(self):
 
         related: https://pagure.io/freeipa/issue/8082
         """
-        tasks.install_client(self.master, self.clients[0],
-                             extra_args=['--ssh-trust-dns'])
-        result = self.clients[0].run_command(['cat', '/etc/ssh/ssh_config'])
-        assert 'HostKeyAlgorithms' not in result.stdout_text
+        try:
+            tasks.install_client(self.master, self.clients[0],
+                                 extra_args=['--ssh-trust-dns'])
+            result = self.clients[0].run_command(['cat', '/etc/ssh/ssh_config'])
+            assert 'HostKeyAlgorithms' not in result.stdout_text
+        finally:
+            tasks.uninstall_client(self.clients[0])
+
+
+class TestClientInstallParams(IntegrationTest):
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True)
+
+    def test_client_install_ntp_params(self):
+        """
+        NTP(d) configuration parameters
+        """
+        client = self.clients[0]
+        tasks.install_client(self.master, client,
+                             extra_args=['--no-ntp'])
+        try:
+            result = client.run_command(['systemctl',
+                                         'status',
+                                         'ntpd'])
+            assert result.returncode == 4  # service not found
+        finally:
+            tasks.uninstall_client(client)
```

Comment:
Won't the IntegrationTest uninstall classmethod handle calling tasks.uninstall_client?

Reply:
Hello @rcritten, thank you for your question. Yes, it should. However, when I omit manual clean-up, subsequent tests fail with "already configured". I'll investigate further. Stay tuned!